packages/devtools/tests/__snapshots__/launcher.test.ts.snap

```diff
@@ -208,6 +250,7 @@ Array [
       ],
       "chromePath": undefined,
       "ignoreDefaultFlags": true,
+      "port": undefined,
```

Comment:
This should fail because the port property is not applied anymore with the latest change, can you run the tests yourself? It should fail.

Reply:
@christian-bromann Thank you.  I removed this and the other ones I had earlier inserted.  I cleared out node_modules, re-ran the setup-full npm script, and now everything is passing.